.travis.yml

```diff
@@ -1,3 +1,14 @@
 language: erlang
 otp_release:
-   - 17.3
+  - 18.1
```

Comment:
Travis only supports up to 18.2.1: https://docs.travis-ci.com/user/languages/erlang -- I can bump it to that.